### PR TITLE
Generalize listing page styles

### LINKED ISSUE #375: Generalize CSS classes for detail tables
1. Make a general set of CSS classes for `featuresetView` and `datasetView`.
2. Apply these classes to `studyView`, to make it use the new sectioned table layout.

### PR DESCRIPTION
**Pull request by msmolens**
Fixes #375.

### CODE REVIEW

**Comment by brianhelba** [comment]
Any reason not to outright delete `web_external/Datasets/datasetPage.styl`?

File: web_external/Studies/studyPage.jade [hunk 1]
```diff
@@ -1,37 +1,46 @@
...
-.isic-study-featureset-container
-  h4 Featureset
-  .isic-listing-content.isic-study-featureset-content
-    = study.featureset().name()
+  table.table.table-condensed.isic-listing-item-table
+    tr
+      td.isic-listing-item-table-section-header Users
+        |  (#{study.get('users').length})
+        if studyAdmin
+          a.isic-study-add-user-button.btn.btn-default.btn-sm.isic-tooltip(
```
Comment (brianhelba) [inline web_external/Studies/studyPage.jade]: This button looks a little crowded. Using `btn-xs` is slightly better, but not great. Any ideas?

File: web_external/Studies/studyPage.styl [hunk 2]
```diff
@@ -9,6 +9,8 @@
 
 .isic-study-users-content
   ul
+    list-style none
```
Comment (brianhelba) [inline web_external/Studies/studyPage.styl]: In my own subjective opinion, this list is still pretty ugly, but re-adding the bullets doesn't seem to help. I'm fine with leaving it alone for now, but any future improvements are welcome.

**Comment by brianhelba** [comment]
@msmolens Review finished. Awaiting your feedback on two items.

File: web_external/Studies/studyPage.jade (hunk 1 above)
Comment (msmolens) [inline web_external/Studies/studyPage.jade]: `btn-xs` looks better. I'm not sure how else to improve it right now.

**Comment by msmolens** [comment]
I like having the three types of files for a view in place (.js, .jade, .styl) even if the Stylus file is empty, but I can delete it if you think that's better. Note that there are some other 0-byte .styl files.

**Comment by brianhelba** [comment]
@msmolens No problem, empty files are fine if they're intended.

Once we move to 2.0 and have to explicitly include Stylus files, it might be nice to be more uniform about always having a file available.

**Comment by brianhelba** [comment]
Tests were already passing. Gonna merge.

**Pull request merged**